assets/elasticsearch/methodology.md

```
@@ -0,0 +1,26 @@
+
+## Basics
+
+Version: [8.6.2][download]
+
+## Setup
+
+We deploy [elasticsearch] in a single-node configuration.
+
+## Specifics
+
+We disable the security feature of [xpack][disabling-xpack]. We use Elasticsearch's Python package for data ingestion 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)